content/en/tracing/trace_collection/compatibility/java.md

```diff
@@ -409,3 +410,4 @@ The solution to this issue is to upgrade to version 4.6.0 or later.
 [5]: /tracing/trace_collection/otel_instrumentation/java/
 [7]: https://www.graalvm.org/downloads/
 [9]: /tracing/trace_explorer/
+[10]: /opentelemetry/interoperability/instrumentation_libraries/
```

Comment:
Can we have this use the Java tab by default?

Reply:
Yes, I have this locally but I haven't pushed the latest batch of edits